### PR TITLE
Stop failing silently if cloc is not installed

### PR DESCRIPTION
**Pull request by SRS-WRKS**
Stops silent failure of setup script if croc is not present on build machine.

### CODE REVIEW

File: code/importers/cloc/setup.sh [hunk 1]
```diff
@@ -1,4 +1,6 @@
 cd "$(dirname "$0")"
+command -v croc ||  echo "Install cloc and rerun"
```
Comment (breck7) [inline code/importers/cloc/setup.sh:2]: croc? did you mean cloc?

File: code/importers/cloc/setup.sh [hunk 2]
```diff
@@ -1,4 +1,6 @@
...
+command -v croc ||  exit 125
```
Comment (breck7) [inline code/importers/cloc/setup.sh:3]: what does this line do?

File: code/importers/cloc/setup.sh (hunk 1 above)
Comment (breck7) [reply code/importers/cloc/setup.sh:2]: @SRS-WRKS

File: code/importers/cloc/setup.sh (hunk 2 above)
Comment (breck7) [reply code/importers/cloc/setup.sh:3]: @SRS-WRKS

Comment (SRS-WRKS) [reply code/importers/cloc/setup.sh:3]: Checks to see if program cloc is installed on the build machine.

File: code/importers/cloc/setup.sh (hunk 1 above)
Comment (SRS-WRKS) [reply code/importers/cloc/setup.sh:2]: YES. I am sorry. cloc. It is is an embarrassing typo .

**Pull request closed without merging**